pg_bm25/src/env.rs

```diff
@@ -0,0 +1,127 @@
+use once_cell::sync::Lazy;
+use pgrx::{register_xact_callback, PgXactCallbackEvent};
+use std::{
+    collections::HashSet,
+    ffi::CStr,
+    panic::{RefUnwindSafe, UnwindSafe},
+    path::PathBuf,
+    sync::{Arc, Mutex, PoisonError},
```

Comment:
`parking_lot::Mutex` claims to be faster. Not sure if it makes a difference for us but thought I'd bring it up.

Reply:
It's not really necessary here, this is only used for the once-per-transaction callback.